.bazelrc

```diff
@@ -1,12 +1,17 @@
 try-import %workspace%/.bazelrc.user
 
-build --symlink_prefix=_bazel/ --ui_event_filters=-DEBUG --define gotags=bazel,gss --experimental_proto_descriptor_sets_include_source_info --incompatible_strict_action_env --incompatible_enable_cc_toolchain_resolution
-test --config=test --experimental_ui_max_stdouterr_bytes=10485760
+build --define gotags=bazel,gss
+build --experimental_proto_descriptor_sets_include_source_info
+build --incompatible_strict_action_env --incompatible_enable_cc_toolchain_resolution
+build --symlink_prefix=_bazel/ --experimental_no_product_name_out_symlink
+test --config=test --experimental_ui_max_stdouterr_bytes=10485760 --experimental_no_product_name_out_symlink
 build:with_ui --define cockroach_with_ui=y
 build:test --define crdb_test=y
 build:race --@io_bazel_rules_go//go/config:race --test_env=GORACE=halt_on_error=1 --test_sharding_strategy=disabled
 test:test --test_env=TZ=
 test:race --test_timeout=1200,6000,18000,72000
+
+build --ui_event_filters=-DEBUG
 query --ui_event_filters=-DEBUG
```

Comment:
Not sure whether `query` needs `--experimental_no_product_name_out_symlink` as well?

Reply:
```
INFO: Reading rc options for 'query' from /Users/irfansharif/Software/src/github.com/cockroachdb/cockroach/.bazelrc:
  'query' options: --ui_event_filters=-DEBUG --experimental_no_product_name_out_symlink
ERROR: --experimental_no_product_name_out_symlink :: Unrecognized option: --experimental_no_product_name_out_symlink
```
Nah.